db/migrate/20180924124835_create_rents.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rents do |t|
+      t.references :user, foreign_key: true
+      t.references :book, foreign_key: true
+      t.date :start_date
```

Comment:
Since `start_date` and `end_date` are required fields, don't forget to add the null: false statement.
Also don't forget to validate the presence in the model.